projects/ngqp-demo/src/app/shared/docs-page.pipes.ts

```diff
@@ -21,6 +21,8 @@ export class DocsPageNamePipe implements PipeTransform {
                 return 'Working with the Model';
             case DocsPage.GLOBAL_CONFIGURATION:
                 return 'Global Configuration';
+            case DocsPage.STANDALONE_QUERY_PARAM:
+                return 'Standalone Query Param';
```

Comment:
"Query Param" is a name we only use to refer to the class (and then it should be QueryParam without space), but I'd just use "Standalone Parameters" here

Reply:
Fixed.